# Encuesta Permanente de Hogares (EPH)

#### Segundo trimestre del año 2025

- Los resultados de la EPH están referidos a la población de los Aglomerados Urbanos, no a toda la población de la provincia ni del país.
- El análisis se centrará sobre la población total estimada del Aglomerado Urbano N° 93 (Viedma-Carmen de Patagones).


# Variables del Dataset:

**individual**:

| Variable       | Descripción                                             |
| -------------- | ------------------------------------------------------- |
| **AGLOMERADO** | Código del aglomerado urbano donde reside la persona.   |
| **CODUSU**     | Identificador único del hogar dentro de la EPH.         |
| **NRO_HOGAR**  | Número del hogar dentro de la vivienda seleccionada.    |
| **COMPONENTE** | Número de persona dentro del hogar.                     |
| **CH04**       | Sexo de la persona.                                     |
| **CH05**       | Edad de la persona.                                     |
| **CH06**       | Parentesco con el jefe/a del hogar.                     |
| **ESTADO**     | Condición de actividad (ocupado, desocupado, inactivo). |
| **CAT_OCUP**   | Categoría ocupacional del trabajo principal.            |
| **PP07H**      | Horas trabajadas en la ocupación principal.             |
| **P21**        | Ingreso laboral de la ocupación principal.              |
| **P47T**       | Ingreso total individual.                               |
| **ANO4**       | Año del relevamiento.                                   |
| **TRIMESTRE**  | Trimestre del relevamiento.                             |
| **IPCF**       | Monto del ingreso per cápita familiar                   |

**hogar**:

| Variable       | Descripción                                                      |
| -------------- | ---------------------------------------------------------------- |
| **AGLOMERADO** | Código del aglomerado urbano del hogar.                          |
| **CODUSU**     | Identificador único del hogar dentro de la EPH.                  |
| **NRO_HOGAR**  | Número del hogar dentro de la vivienda seleccionada.             |
| **CANT_PER**   | Cantidad total de personas que integran el hogar.                |
| **ITF**        | Ingreso total familiar (clave para medición de pobreza).         |
| **PONDERA**    | Factor de expansión del hogar (peso muestral).                   |
| **PONDIH**     | Ponderador ajustado por no respuesta de ingresos (para pobreza). |
| **REGION**     | Región estadística del hogar.                                    |
| **TIPO_HOGAR** | Tipo de hogar según composición familiar.                        |
| **ANO4**       | Año del relevamiento.                                            |
| **TRIMESTRE**  | Trimestre del relevamiento.                                      |
| **II7**        | La condición de vivienda. Si es propitaria la familia o no.      |


#### Aclaraciones metodológicas:

| Ponderador | Aplica a Variables | ¿Qué Mide o Cuenta? |
|---|---|---|
| PONDER | Todo lo que NO es ingreso (edad, ocupación, sexo, vivienda) | La población total o cantidad de hogares por características no monetarias |
| PONDII | Ingreso Total Individual (`P47T`) y no laborales | El ingreso total de las personas |
| PONDIIO | Ingreso de la Ocupación Principal (`P21`) | El ingreso de las personas solo por su trabajo principal |
| PONDIH | Ingreso Total Familiar (`ITF`, `IPCF`) | La población sujeta a análisis de pobreza y distribución del ingreso (del hogar) |

In [4]:
# definimos el directorio de trabajo
import os
path = r'C:\PROYECTOS\SQL\Practica-INDEC\EPH_usu_2_Trim_2025_xls'
os.chdir(path)
os.getcwd()

'C:\\PROYECTOS\\SQL\\Practica-INDEC\\EPH_usu_2_Trim_2025_xls'

In [5]:
# revisamos los archivos que se encuentran en el directorio
os.listdir()

['EPH_2025_T2.db', 'usu_hogar_T225.xlsx', 'usu_individual_T225.xlsx']

In [6]:
# instalamos la extension
#!pip install ipython-sql

# cargamos la funcion magina sql
%load_ext sql

# librerias para visualizaciones 
import seaborn as sns
import matplotlib.pyplot as plt

In [7]:
import pandas as pd
# importamos el modulo sqlite para manejar bases de datos SQLite
import sqlite3

In [8]:
# conexion a SQLite
con = sqlite3.connect('EPH_2025_T2.db')
cur = con.cursor()

In [6]:
# cargar los archivos con pandas
df_hogar = pd.read_excel('usu_hogar_T225.xlsx')
df_individual = pd.read_excel('usu_individual_T225.xlsx')

# guardar en SQLite en tablas separadas
df_hogar.to_sql('hogar',con, if_exists='replace',index=False)
df_individual.to_sql('individual',con, if_exists='replace',index=False)


46086

In [9]:
# conecta a la base datos
%sql sqlite:///EPH_2025_T2.db

In [10]:
import prettytable
prettytable.DEFAULT = 'DEFAULT'

#### Verificamos que la creacion de las tablas haya sido exitosa realizando consultas simples

In [11]:
%sql SELECT * FROM hogar LIMIT 5;

 * sqlite:///EPH_2025_T2.db
Done.


[('TQRMNOPUSHMOLQCDEFLID00881764', 2025, 2, 1, 1, 43, 'S', 5, 522, 1, None, 4, 1, None, 4, 1, 1, 1, None, 1, 1, 1, 1, 2, 2, 2, 4, 2, 2, 0, 1, 1, 1, 2, 0, 2, 0, 1, None, 1, None, 1, 1, 2, 0, 0, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 4, 0, 4, 720000, 3, 3.0, 3, 3.0, None, 3, 180000.0, 2, 2.0, 1, 2.0, None, 2, 1720, 2, 0, 4, 0, 0, 0, 1, 2, 2, 2, 2, 0),
 ('TQRMNOQRPHKOLTCDEFLID00881765', 2025, 2, 1, 1, 43, 'S', 5, 346, 1, None, 3, 1, None, 4, 1, 1, 1, None, 1, 1, 1, 2, 2, 2, 2, 3, 2, 2, 0, 1, 2, 2, 2, 0, 2, 0, 1, None, 2, None, 1, 1, 2, 0, 0, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 1, 2, 2, 2, 2, 3, 0, 3, 0, 12, 12.0, 12, 12.0, None, 12, 0.0, 12, 12.0, 12, 12.0, None, 12, 0, 2, 0, 1, 0, 0, 0, 2, 2, 2, 2, 2, 0),
 ('TQRMNOSTUHMMKOCDEIGED00879108', 2025, 2, 1, 1, 43, 'N', 30, 163, 1, None, 3, 1, None, 2, 1, 1, 1, None, 1, 1, 1, 1, 2, 1, 2, 3, 2, 1, 1, 1, 2, 2, 2, 0, 2, 0, 6, None, 1, None, 1, 1, 2, 0, 0, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 1, 2, 2, 2, 1, 2, 2, 2, 2, 0, 2, 700000, 3, 3.0, 3, None, 3.0, 3, 350000.0, 4, 4.0, 4, None, 4.0, 3, 220, 1, 2, 97, 0, 0, 0, 2, 2, 2, 2, 2, 0),
 ('TQRMNOQWYHKMKNCDEIGED00857021', 2025, 2, 1, 1, 43, 'N', 30, 104, 1, None, 3, 1, None, 1, 1, 1, 1, None, 1, 1, 1, 1, 2, 2, 2, 3, 2, 2, 0, 1, 2, 1, 2, 0, 2, 0, 1, None, 1, None, 1, 1, 2, 0, 0, 2, 2, 2, 2, 1, 2, 2, 2, 2, 2, 2, 2, 2, 2, 1, 2, 2, 2, 2, 2, 2, 2, 3, 0, 3, 456000, 2, 2.0, 2, None, 2.0, 2, 152000.0, 1, 1.0, 1, None, 1.0, 1, 117, 1, 0, 3, 0, 0, 0, 2, 2, 2, 2, 2, 0),
 ('TQRMNOPRTHMMKRCDEIGED00878749', 2025, 2, 1, 1, 43, 'N', 30, 131, 1, None, 3, 1, None, 1, 1, 1, 2, None, 1, 1, 1, 1, 2, 2, 2, 3, 2, 2, 0, 1, 2, 2, 2, 0, 2, 0, 1, None, 1, None, 1, 2, 1, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 1, 0, 1, 400000, 1, 1.0, 1, None, 1.0, 1, 400000.0, 5, 5.0, 4, None, 5.0, 3, 130, 1, 0, 96, 0, 0, 0, 2, 2, 2, 2, 2, 1)]

In [12]:
%sql SELECT * FROM individual LIMIT 5;

 * sqlite:///EPH_2025_T2.db
Done.


[('TQRMNOQWSHMKMPCDEIIAD00878792', 2025, 2, 1, 2, 1, 1, 'S', 32, 843, 2, 2, '1900-01-01 00:00:00', 49, 1, 1.0, 1, 2, 0, 2, 1, None, 1, None, 1, None, 2, 1, 3, 0, None, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1.0, 0.0, 35.0, 0.0, 2.0, 0.0, 2.0, 2.0, 2.0, 1.0, 8501.0, 2.0, 0.0, 0.0, 0.0, 0.0, 99.0, 9.0, 56314.0, 11.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 6.0, 2.0, 0.0, 0.0, 2.0, 2.0, 2.0, 2.0, 5.0, 1.0, 1.0, 1.0, 1.0, 0.0, 1.0, 0.0, 1.0, 1.0, 800000.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, None, 0.0, 0.0, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, 800000, 6, None, 6, 6.0, None, 4, 1266, 0, 800000.0, 7, None, 6, 7.0, None, 5, 1223, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1400000, 6, None, 6, 6.0, None, 5, 466666.67, 6, None, 5, 5.0, None, 3, 1618, 0, 0, 0, 0, 2.0, 1.0, 1.0, 0, 0, 0, 0, 0, 0.0, 9.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 2.0, 2.0, 2.0, 0.0, 0.0, 0.0, 1.0, 0.0, 1.0, 5.0, 0.0, 7.0, None, None, None, None, None, None, None, None, None, None, None, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 6.0, 6.0, 6.0, None, None, 4.0),
 ('TQRMNOQWSHMKMPCDEIIAD00878792', 2025, 2, 1, 3, 1, 1, 'S', 32, 843, 3, 2, '1900-01-01 00:00:00', 22, 5, 4.0, 1, 2, 0, 4, 1, None, 1, None, 1, None, 4, 1, 3, 0, None, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1.0, 0.0, 72.0, 0.0, 2.0, 0.0, 2.0, 2.0, 3.0, 2.0, 4803.0, 2.0, 0.0, 0.0, 0.0, 0.0, 3.0, 0.0, 30313.0, 11.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 9.0, 9.0, 4.0, 2.0, 2.0, 2.0, 2.0, 5.0, 2.0, 2.0, 2.0, 2.0, 5.0, 2.0, 2.0, 1.0, 4.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, None, 0.0, 0.0, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, 0, 0, None, 0, 0.0, None, 0, 843, 0, 0.0, 0, None, 0, 0.0, None, 0, 843, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1400000, 6, None, 6, 6.0, None, 5, 466666.67, 6, None, 5, 5.0, None, 3, 1618, 0, 0, 0, 0, 2.0, 2.0, 9.0, 0, 0, 0, 0, 0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 2.0, 2.0, 2.0, 9.0, 9.0, 9.0, 0.0, 0.0, 1.0, 6.0, 0.0, 12.0, None, None, None, None, None, None, None, None, None, None, None, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 6.0, 6.0, 6.0, None, None, 4.0),
 ('TQRMNOQUVHMLMQCDEIIAD00878965', 2025, 2, 1, 1, 1, 1, 'S', 32, 117, 1, 1, '1971-05-05 00:00:00', 54, 1, 4.0, 1, 2, 0, 4, 1, None, 1, None, 1, None, 4, 2, 3, 0, None, 2, 1, 2, 2, 2, 1, 2, 2, 0, 0, 0, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, 4.0, 1.0, 0.0, 2.0, 2.0, 9700.0, 1.0, 2.0, 2.0, 1.0, 0.0, 0.0, 47313.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 2.0, 4.0, 0.0, 0.0, 0.0, 0.0, 2.0, 0, 0, None, 0, 0.0, None, 0, 117, 0, 0.0, 0, None, 0, 0.0, None, 0, 117, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 12, None, 12, 12.0, None, 12, 0.0, 12, None, 12, 12.0, None, 12, 0, 0, 0, 0, 0, None, None, None, 1, 1, 0, 0, 0, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, 2.0, 2.0, 1.0, 2.0, 1.0, 2.0, 2.0, 2.0, 2.0, 2.0, 0.0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, None, None, None, None, None, None),
 ('TQRMNOQUVHMLMQCDEIIAD00878965', 2025, 2, 1, 2, 1, 1, 'S', 32, 117, 6, 1, '1938-03-04 00:00:00', 87, 4, 1.0, 1, 2, 0, 4, 1, None, 1, None, 1, None, 4, 3, 0, 1, None, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2, 2, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, No

In [13]:
# consultar las tablas que contine la base de datos
%sql SELECT name FROM sqlite_master WHERE type='table';

 * sqlite:///EPH_2025_T2.db
Done.


[('hogar',), ('individual',)]

In [14]:
# eliminamos las tablas duplicadas por error
%sql DROP TABLE IF EXISTS usu_hogar_T225;
%sql DROP TABLE IF EXISTS usu_individual;

 * sqlite:///EPH_2025_T2.db
Done.
 * sqlite:///EPH_2025_T2.db
Done.


[]

In [15]:
# verificamos
%sql SELECT name FROM sqlite_master WHERE type='table';

 * sqlite:///EPH_2025_T2.db
Done.


[('hogar',), ('individual',)]

In [16]:
# ver la estructura de las tablas
%sql PRAGMA table_info([hogar]);

 * sqlite:///EPH_2025_T2.db
Done.


[(0, 'CODUSU', 'TEXT', 0, None, 0),
 (1, 'ANO4', 'INTEGER', 0, None, 0),
 (2, 'TRIMESTRE', 'INTEGER', 0, None, 0),
 (3, 'NRO_HOGAR', 'INTEGER', 0, None, 0),
 (4, 'REALIZADA', 'INTEGER', 0, None, 0),
 (5, 'REGION', 'INTEGER', 0, None, 0),
 (6, 'MAS_500', 'TEXT', 0, None, 0),
 (7, 'AGLOMERADO', 'INTEGER', 0, None, 0),
 (8, 'PONDERA', 'INTEGER', 0, None, 0),
 (9, 'IV1', 'INTEGER', 0, None, 0),
 (10, 'IV1_ESP', 'TEXT', 0, None, 0),
 (11, 'IV2', 'INTEGER', 0, None, 0),
 (12, 'IV3', 'INTEGER', 0, None, 0),
 (13, 'IV3_ESP', 'TEXT', 0, None, 0),
 (14, 'IV4', 'INTEGER', 0, None, 0),
 (15, 'IV5', 'INTEGER', 0, None, 0),
 (16, 'IV6', 'INTEGER', 0, None, 0),
 (17, 'IV7', 'INTEGER', 0, None, 0),
 (18, 'IV7_ESP', 'TEXT', 0, None, 0),
 (19, 'IV8', 'INTEGER', 0, None, 0),
 (20, 'IV9', 'INTEGER', 0, None, 0),
 (21, 'IV10', 'INTEGER', 0, None, 0),
 (22, 'IV11', 'INTEGER', 0, None, 0),
 (23, 'IV12_1', 'INTEGER', 0, None, 0),
 (24, 'IV12_2', 'INTEGER', 0, None, 0),
 (25, 'IV12_3', 'INTEGER', 0, None, 0),
 (26, 'II1', 'INTEGER', 0, None, 0),
 (27, 'II2', 'INTEGER', 0, None, 0),
 (28, 'II3', 'INTEGER', 0, None, 0),
 (29, 'II3_1', 'INTEGER', 0, None, 0),
 (30, 'II4_1', 'INTEGER', 0, None, 0),
 (31, 'II4_2', 'INTEGER', 0, None, 0),
 (32, 'II4_3', 'INTEGER', 0, None, 0),
 (33, 'II5', 'INTEGER', 0, None, 0),
 (34, 'II5_1', 'INTEGER', 0, None, 0),
 (35, 'II6', 'INTEGER', 0, None, 0),
 (36, 'II6_1', 'INTEGER', 0, None, 0),
 (37, 'II7', 'INTEGER', 0, None, 0),
 (38, 'II7_ESP', 'TEXT', 0, None, 0),
 (39, 'II8', 'INTEGER', 0, None, 0),
 (40, 'II8_ESP', 'TEXT', 0, None, 0),
 (41, 'II9', 'INTEGER', 0, None, 0),
 (42, 'V1', 'INTEGER', 0, None, 0),
 (43, 'V2', 'INTEGER', 0, None, 0),
 (44, 'V2_02', 'INTEGER', 0, None, 0),
 (45, 'V2_03', 'INTEGER', 0, None, 0),
 (46, 'V21_01', 'INTEGER', 0, None, 0),
 (47, 'V22_01', 'INTEGER', 0, None, 0),
 (48, 'V3', 'INTEGER', 0, None, 0),
 (49, 'V4', 'INTEGER', 0, None, 0),
 (50, 'V5_02', 'INTEGER', 0, None, 0),
 (51, 'V5_03', 'INTEGER', 0, None, 0),
 (52, 'V6', 'INTEGER', 0, None, 0),
 (53, 'V7', 'INTEGER', 0, None, 0),
 (54, 'V8', 'INTEGER', 0, None, 0),
 (55, 'V9', 'INTEGER', 0, None, 0),
 (56, 'V10', 'INTEGER', 0, None, 0),
 (57, 'V11_01', 'INTEGER', 0, None, 0),
 (58, 'V11_02', 'INTEGER', 0, None, 0),
 (59, 'V12', 'INTEGER', 0, None, 0),
 (60, 'V13', 'INTEGER', 0, None, 0),
 (61, 'V14', 'INTEGER', 0, None, 0),
 (62, 'V15', 'INTEGER', 0, None, 0),
 (63, 'V16', 'INTEGER', 0, None, 0),
 (64, 'V17', 'INTEGER', 0, None, 0),
 (65, 'V18', 'INTEGER', 0, None, 0),
 (66, 'V19_A', 'INTEGER', 0, None, 0),
 (67, 'V19_B', 'INTEGER', 0, None, 0),
 (68, 'IX_TOT', 'INTEGER', 0, None, 0),
 (69, 'IX_MEN10', 'INTEGER', 0, None, 0),
 (70, 'IX_MAYEQ10', 'INTEGER', 0, None, 0),
 (71, 'ITF', 'INTEGER', 0, None, 0),
 (72, 'DECIFR', 'INTEGER', 0, None, 0),
 (73, 'IDECIFR', 'REAL', 0, None, 0),
 (74, 'RDECIFR', 'INTEGER', 0, None, 0),
 (75, 'GDECIFR', 'REAL', 0, None, 0),
 (76, 'PDECIFR', 'REAL', 0, None, 0),
 (77, 'ADECIFR', 'INTEGER', 0, None, 0),
 (78, 'IPCF', 'REAL', 0, None, 0),
 (79, 'DECCFR', 'INTEGER', 0, None, 0),
 (80, 'IDECCFR', 'REAL', 0, None, 0),
 (81, 'RDECCFR', 'INTEGER', 0, None, 0),
 (82, 'GDECCFR', 'REAL', 0, None, 0),
 (83, 'PDECCFR', 'REAL', 0, None, 0),
 (84, 'ADECCFR', 'INTEGER', 0, None, 0),
 (85, 'PONDIH', 'INTEGER', 0, None, 0),
 (86, 'VII1_1', 'INTEGER', 0, None, 0),
 (87, 'VII1_2', 'INTEGER', 0, None, 0),
 (88, 'VII2_1', 'INTEGER', 0, None, 0),
 (89, 'VII2_2', 'INTEGER', 0, None, 0),
 (90, 'VII2_3', 'INTEGER', 0, None, 0),
 (91, 'VII2_4', 'INTEGER', 0, None, 0),
 (92, 'V5_01', 'INTEGER', 0, None, 0),
 (93, 'V21_02', 'INTEGER', 0, None, 0),
 (94, 'V21_03', 'INTEGER', 0, None, 0),
 (95, 'V22_02', 'INTEGER', 0, None, 0),
 (96, 'V22_03', 'INTEGER', 0, None, 0),
 (97, 'V2_01', 'INTEGER', 0, None, 0)]

In [17]:
%sql PRAGMA table_info([individual]);

 * sqlite:///EPH_2025_T2.db
Done.


[(0, 'CODUSU', 'TEXT', 0, None, 0),
 (1, 'ANO4', 'INTEGER', 0, None, 0),
 (2, 'TRIMESTRE', 'INTEGER', 0, None, 0),
 (3, 'NRO_HOGAR', 'INTEGER', 0, None, 0),
 (4, 'COMPONENTE', 'INTEGER', 0, None, 0),
 (5, 'H15', 'INTEGER', 0, None, 0),
 (6, 'REGION', 'INTEGER', 0, None, 0),
 (7, 'MAS_500', 'TEXT', 0, None, 0),
 (8, 'AGLOMERADO', 'INTEGER', 0, None, 0),
 (9, 'PONDERA', 'INTEGER', 0, None, 0),
 (10, 'CH03', 'INTEGER', 0, None, 0),
 (11, 'CH04', 'INTEGER', 0, None, 0),
 (12, 'CH05', 'TIMESTAMP', 0, None, 0),
 (13, 'CH06', 'INTEGER', 0, None, 0),
 (14, 'CH07', 'INTEGER', 0, None, 0),
 (15, 'CH08', 'REAL', 0, None, 0),
 (16, 'CH09', 'INTEGER', 0, None, 0),
 (17, 'CH10', 'INTEGER', 0, None, 0),
 (18, 'CH11', 'INTEGER', 0, None, 0),
 (19, 'CH12', 'INTEGER', 0, None, 0),
 (20, 'CH13', 'INTEGER', 0, None, 0),
 (21, 'CH14', 'REAL', 0, None, 0),
 (22, 'CH15', 'INTEGER', 0, None, 0),
 (23, 'CH15_COD', 'REAL', 0, None, 0),
 (24, 'CH16', 'INTEGER', 0, None, 0),
 (25, 'CH16_COD', 'REAL', 0, None, 0),
 (26, 'NIVEL_ED', 'INTEGER', 0, None, 0),
 (27, 'ESTADO', 'INTEGER', 0, None, 0),
 (28, 'CAT_OCUP', 'INTEGER', 0, None, 0),
 (29, 'CAT_INAC', 'INTEGER', 0, None, 0),
 (30, 'IMPUTA', 'REAL', 0, None, 0),
 (31, 'PP02C1', 'INTEGER', 0, None, 0),
 (32, 'PP02C2', 'INTEGER', 0, None, 0),
 (33, 'PP02C3', 'INTEGER', 0, None, 0),
 (34, 'PP02C4', 'INTEGER', 0, None, 0),
 (35, 'PP02C5', 'INTEGER', 0, None, 0),
 (36, 'PP02C6', 'INTEGER', 0, None, 0),
 (37, 'PP02C7', 'INTEGER', 0, None, 0),
 (38, 'PP02C8', 'INTEGER', 0, None, 0),
 (39, 'PP02E', 'INTEGER', 0, None, 0),
 (40, 'PP02H', 'INTEGER', 0, None, 0),
 (41, 'PP02I', 'INTEGER', 0, None, 0),
 (42, 'PP03C', 'REAL', 0, None, 0),
 (43, 'PP03D', 'REAL', 0, None, 0),
 (44, 'PP3E_TOT', 'REAL', 0, None, 0),
 (45, 'PP3F_TOT', 'REAL', 0, None, 0),
 (46, 'PP03G', 'REAL', 0, None, 0),
 (47, 'PP03H', 'REAL', 0, None, 0),
 (48, 'PP03I', 'REAL', 0, None, 0),
 (49, 'PP03J', 'REAL', 0, None, 0),
 (50, 'INTENSI', 'REAL', 0, None, 0),
 (51, 'PP04A', 'REAL', 0, None, 0),
 (52, 'PP04B_COD', 'REAL', 0, None, 0),
 (53, 'PP04B1', 'REAL', 0, None, 0),
 (54, 'PP04B2', 'REAL', 0, None, 0),
 (55, 'PP04B3_MES', 'REAL', 0, None, 0),
 (56, 'PP04B3_ANO', 'REAL', 0, None, 0),
 (57, 'PP04B3_DIA', 'REAL', 0, None, 0),
 (58, 'PP04C', 'REAL', 0, None, 0),
 (59, 'PP04C99', 'REAL', 0, None, 0),
 (60, 'PP04D_COD', 'REAL', 0, None, 0),
 (61, 'PP04G', 'REAL', 0, None, 0),
 (62, 'PP05B2_MES', 'REAL', 0, None, 0),
 (63, 'PP05B2_ANO', 'REAL', 0, None, 0),
 (64, 'PP05B2_DIA', 'REAL', 0, None, 0),
 (65, 'PP05C_1', 'REAL', 0, None, 0),
 (66, 'PP05C_2', 'REAL', 0, None, 0),
 (67, 'PP05C_3', 'REAL', 0, None, 0),
 (68, 'PP05E', 'REAL', 0, None, 0),
 (69, 'PP05F', 'REAL', 0, None, 0),
 (70, 'PP05H', 'REAL', 0, None, 0),
 (71, 'PP06A', 'REAL', 0, None, 0),
 (72, 'PP06C', 'REAL', 0, None, 0),
 (73, 'PP06D', 'REAL', 0, None, 0),
 (74, 'PP06E', 'REAL', 0, None, 0),
 (75, 'PP06H', 'REAL', 0, None, 0),
 (76, 'PP07A', 'REAL', 0, None, 0),
 (77, 'PP07C', 'REAL', 0, None, 0),
 (78, 'PP07D', 'REAL', 0, None, 0),
 (79, 'PP07E', 'REAL', 0, None, 0),
 (80, 'PP07F1', 'REAL', 0, None, 0),
 (81, 'PP07F2', 'REAL', 0, None, 0),
 (82, 'PP07F3', 'REAL', 0, None, 0),
 (83, 'PP07F4', 'REAL', 0, None, 0),
 (84, 'PP07F5', 'REAL', 0, None, 0),
 (85, 'PP07G1', 'REAL', 0, None, 0),
 (86, 'PP07G2', 'REAL', 0, None, 0),
 (87, 'PP07G3', 'REAL', 0, None, 0),
 (88, 'PP07G4', 'REAL', 0, None, 0),
 (89, 'PP07G_59', 'REAL', 0, None, 0),
 (90, 'PP07H', 'REAL', 0, None, 0),
 (91, 'PP07I', 'REAL', 0, None, 0),
 (92, 'PP07J', 'REAL', 0, None, 0),
 (93, 'PP07K', 'REAL', 0, None, 0),
 (94, 'PP08D1', 'REAL', 0, None, 0),
 (95, 'PP08D4', 'REAL', 0, None, 0),
 (96, 'PP08F1', 'REAL', 0, None, 0),
 (97, 'PP08F2', 'REAL', 0, None, 0),
 (98, 'PP08J1', 'REAL', 0, None, 0),
 (99, 'PP08J2', 'REAL', 0, None, 0),
 (100, 'PP08J3', 'REAL', 0, None, 0),
 (101, 'PP09A', 'REAL', 0, None, 0),
 (102, 'PP09A_ESP', 'TEXT', 0, None, 0),
 (103, 'PP09B', 'REAL', 0, None, 0),
 (104, 'PP09C', 'REAL', 0, None, 0),
 (105, 

# HOGAR

#### Cantidad de filas

In [12]:
%sql SELECT COUNT(*) \
    FROM hogar;

 * sqlite:///EPH_2025_T2.db
Done.


[(16169,)]

#### Cantidad de muestras realizadas por hogares en el Aglomerado N° 93 (Viedma-Carmen de Patagones)

In [13]:
%%sql

-- Contar los hogares entrevistados

SELECT COUNT(*)
FROM hogar
WHERE AGLOMERADO = 93;

 * sqlite:///EPH_2025_T2.db
Done.


[(327,)]

#### ¿Cuál es el número total estimado de hogares en el aglomerado urbano Viedma-Patagones?

In [15]:
%sql SELECT SUM(PONDERA) \
    FROM hogar \
    WHERE AGLOMERADO =93;

 * sqlite:///EPH_2025_T2.db
Done.


[(34323,)]

#### De numero total estimado de hogares en el alomerado urbano Viedma-Patagones, ¿cuántos de esos pertenece a familias propitarias y cuántas a familias inquilinas?

In [16]:
%%sql

SELECT 
    SUM(CASE WHEN II7 IN (1, 2) THEN PONDIH ELSE 0 END) AS hogares_propietarios_estimados,
    SUM(CASE WHEN II7 = 3 THEN PONDIH ELSE 0 END) AS hogares_inquilinos_estimados
FROM hogar
WHERE AGLOMERADO = 93;

 * sqlite:///EPH_2025_T2.db
Done.


[(23337, 7646)]

### ¿Cuántas familias estimadas perciben un ingreso mayor o igual a 3 000 000 de pesos?

In [35]:
%%sql

SELECT SUM(PONDIH)
FROM hogar
WHERE AGLOMERADO = 93
AND ITF >= 3000000;

 * sqlite:///EPH_2025_T2.db
Done.


[(1091,)]

#### De lo que perciben un ingreso superior a los 3 000 000 millones de pesos, ¿cuántas familias son propietadas?

In [36]:
%%sql

SELECT SUM(PONDIH) AS familias_propietarias_altos_ingresos
FROM hogar
WHERE AGLOMERADO = 93 AND ITF >=3000000 AND II7 IN (1,2);


 * sqlite:///EPH_2025_T2.db
Done.


[(1091,)]

## Resultados parciales

* Coincide que, aquellas familias que tienen un ingreso considerado "superior" al momento del análisis, son las mismas familias que son propietarias. 

# Familias pobres

### Referencia CBT (Canasta Básica Total)

Para un análisis con rigor regional en el Aglomerado 93 (Viedma-Carmen de Patagones, Región Patagonia), se utiliza la CBT ajustada:

-  **Fuente Oficial:** **INDEC** - Valorización mensual de la Canasta Básica Total (Informe de Junio de 2025).
-  **Link de referencia:** [https://www.indec.gob.ar/uploads/informesdeprensa/canasta\_07\_25B95CBBB21C.pdf](https://www.indec.gob.ar/uploads/informesdeprensa/canasta_07_25B95CBBB21C.pdf)
- **Metodología de Ajuste (4 Personas):** El valor es el resultado de multiplicar la CBT del Adulto Equivalente de la Patagonia por **3.09** (el total de Unidades Consumidoras para este tipo de hogar).
- **Valor de Corte Utilizado (CBT H-4 Patagonia):** **\$1.128.895**

#### De los 34323 hogares estimados en el anglomerado urbano de Vieda-Patagones, ¿cuál es el porcentaje que se encuentran por debajo de la línea de pobreza (teniendo en cuenta la CBT)?


In [34]:
%%sql
SELECT
    -- Numerador: Suma de HOGARES de 4 personas que son pobres (ITF < CBT_H4)
    (SUM(CASE WHEN T1.ITF < 1128895 THEN T1.PONDIH ELSE 0 END) * 100.0)
    /
    -- Denominador: Suma del total de HOGARES de 4 personas en la muestra
    SUM(T1.PONDIH) AS Tasa_Pobreza_Hogares_4_Personas_Estimada
FROM
    hogar AS T1
WHERE
    T1.AGLOMERADO = 93
    AND T1.IX_TOT = 4
    AND T1.ITF >= 0;

 * sqlite:///EPH_2025_T2.db
Done.


[(41.43564356435643,)]

## Conclusiones preliminares

* Se estima que: aproximadamente el 41.44 % de los hogares de 4 personas en el aglomerado Viedma-Carmen de Patagones se encuentran por debajo de la línea de pobreza.
  